test/library/calibration/test_rabi.py

```diff
@@ -272,7 +273,10 @@ def test_bad_analysis(self):
         """Test the Rabi analysis."""
         experiment_data = ExperimentData()
 
-        thetas = np.linspace(0.0, np.pi / 4, 31)
+        # Change rotation angle with square root of amplitude so that
+        # population versus amplitude will not be sinusoidal and the fit will
+        # be bad.
+        thetas = np.sqrt(np.linspace(0.0, 4 * np.pi**2, 31))
```

Comment:
It seemed like this test chose somewhat arbitrary input data to be "bad" but changing simulators made it start fitting as good again. I made the data worse so that it should more reliably fit as bad.